docs/samples/browser-plugin-meetings/app.js

```diff
@@ -3780,11 +3791,19 @@ function createMembersTable(members) {
 
     td5.appendChild(label5);
 
+
+    if (isCurrentUser(member)) {
+      td6.appendChild(createButton(member.isBrb ? 'Remove away' : 'Apply away', toggleBrb, {id: 'brb-btn'}));
```

Comment:
I would change the wording here to "Step Away" and "Back to Meeting". That just seems the most clear to me, and having it match the client UI makes it easy to understand that this is mapped to the step away feature.

Reply:
@brycetham I have tried this combination, but I don't like that "Back to Meeting" span on two lines and I can't do a style fix for that easily, I need to change the style for the table, not sure if this is worth that, but I'm open to discuss 

<img width="1156" alt="Screenshot 2024-12-13 at 14 51 28" src="https://github.com/user-attachments/assets/9f9bc824-3d44-4bc8-b236-5490988cb290" />